english/matrix/matrix.md

```diff
@@ -23,5 +23,5 @@ It's dark and fluffy.
 While you don't believe that you might be the One,
  it's only up to you to save the world.
 
-[You crawl through the hole you just made and...] (../matrix2/matrix2.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.